webpack.config.js

```diff
@@ -1,6 +1,6 @@
 const glob = require("glob");
 const path = require("path");
-const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
does this new plugin need to be added to package.json?

Reply:
I don't think so, it is bundled in webpack itself
https://github.com/webpack/webpack/blob/master/package.json#L27